python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -70,6 +70,9 @@ def __shadingSummary( plug ) :
 		if plug[childName]["enabled"].getValue() :
 			info.append( IECore.CamelCase.toSpaced( childName ) + ( " On" if plug[childName]["value"].getValue() else " Off" ) )
 
+	if plug["sssSetName"]["enabled"].getValue() :
+		info.append( "Sss Set Name " + plug["sssSetName"]["value"].getValue() )
```

Comment:
"SSS Set Name", capitalising the acronym bit? Might need to do the same with an explicit label metadata too?